.github/workflows/deno.yml

```diff
@@ -0,0 +1,46 @@
+name: Deno
+
+on:
+  push:
+    paths:
+      - deno/**
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    working-directory: deno
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
```

Comment:
Are you trying to test arm? If there's only one just directly add it.

Reply:
its possible, but I guess no need to run on arm, I updated it as you suggested